app/femr/ui/models/admin/configure/IndexViewModelPost.java

```diff
@@ -4,12 +4,14 @@
 
 public class IndexViewModelPost {
     private List<String> settings;
+    private List<String> descriptions;
 
     public List<String> getSettings() {
         return settings;
     }
 
-    public void setSettings(List<String> settings) {
-        this.settings = settings;
```

Comment:
@judyvbaxter removing this causes an error when you submit the settings. This is supposed to catch the settings during a POST